PYME/Acquire/Hardware/Piezos/piezo_c867.py

```diff
@@ -275,20 +304,15 @@ def _Loop(self):
                         logger.debug('Setting stage target pos: %s' % pos)
                         time.sleep(.01)
                     
-                #check to see if we're on target
-                self.ser_port.write(b'ONT?\n')
-                self.ser_port.flushOutput()
-                time.sleep(0.005)
-                res1 = self.ser_port.readline()
-                ont1 = int(res1.split(b'=')[1]) == 1
-                res1 = self.ser_port.readline()
-                ont2 = int(res1.split(b'=')[1]) == 1
+                # check to see if we're on target
+                # from the manual on 'ONT?' command:
+                #   "The detection of the on-target state is only possible in closed-loop operation (servo mode ON)"
+                # note that the cleaned up on-target check does not use the 'ONT?' command any more
+                if self.servo:
+                    self.onTarget = np.allclose(self.position, self.targetPosition, atol=self.ptol)
+                else: # servo is off
+                    self.onTarget = False                
```

Comment:
Seeing as we're effectively doing soft servoing, I don't think we should be reporting `onTarget=False` if we are on target, even if the servo is disabled (there is no reason to).